src/styles/global/buttons.styl

```diff
@@ -0,0 +1,78 @@
+.button
+  @extends $reset-button
+  border: 2px solid
+  border-radius: 0.5em
```

Comment:
So my comments about using em apply to fonts, paddings, and margins. Stuff that would need to scale. Border radius typically is pixel or percentages since this wouldn't need to scale with zooming and might actually cause some odd radius changes with a zoom.